tutorial/developing_packages/editable_packages.rst

```diff
@@ -0,0 +1,204 @@
+.. _editable_packages:
+
+Packages in editable mode
+=========================
+
+The normal way of working with Conan packages is to run a ``conan create`` or ``conan
+export-pkg`` to store them in the local cache, so that consumers use the packages stored
+in the cache. In some cases, when you want to consume these packages while developing
+them, it can be tedious to run ``conan create`` each time you make changes to the package.
+For those cases, you can put your package in editable mode, and consumers will be able to
+find the headers and artifacts in your local working directory, eliminating the need for
+packaging.
+
+Let's see how we can put a package in editable mode and consume it from the local working
+directory.
+
+Please, first of all, clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/tutorial/developing_packages/editable_packages
+
+There are 2 folders inside this project:
+
+..  code-block:: text
+
+    .
+    ├── hello
+    │   ├── CMakeLists.txt
+    │   ├── conanfile.py
+    │   └── src
+    │       └── hello.cpp
+    └── say
+        ├── CMakeLists.txt
+        ├── conanfile.py
+        ├── include
+        │   └── say.h
+        └── src
+            └── say.cpp
+
+
+- A "say" folder containing a fully fledge package, with its ``conanfile.py``, its source
+  code.
+- A "hello" folder containing a simple consumer project with a ``conanfile.py`` and its
+  source code, which depends on the ``say/1.0`` requirement.
+
+We will put ``say/1.0`` in editable mode and show how the ``hello`` consumer can find ``say/1.0``
+headers and binaries in its local working directory.
+
+Put say/1.0 package in editable mode
+------------------------------------
+
+To avoid creating the package ``say/1.0`` in the cache for every change, we are going to
+put that package in editable mode, creating **a link from the reference in the cache to
+the local working directory**:
+
+.. code-block:: bash
+
+    $ conan editable add say say/1.0
```

Comment:
I think we should change this to make the ref optional with ``--name=say --version=1.0``, same as we do for other commands

Reply:
https://github.com/conan-io/conan/pull/13093

Reply:
great, let's do the same than with the presets thing and make the changes part of the docs for the release to develop2